cg/constants/delivery.py

```diff
@@ -145,6 +145,7 @@
 
 SARSCOV2_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
```

Comment:
Should this not be MUTANT?

Reply:
Just added the tags. Shall I rename to `MUTANT_ANALYSIS_SAMPLE_TAGS` instead?

Seems that this constant is only being used in this repo. Is there another repo that needs to be updated though?

Reply:
That would be great. Only used in this repo. 👍